github/src/main/kotlin/gropius/sync/github/GithubDataService.kt

```diff
@@ -42,6 +76,44 @@ class GithubDataService(
         ).awaitSingle()
     }
 
+    /**
+     * Get a IMSUser for a GitHub user
+     * @param imsProject the project to map the user to
+     * @param user the Jira user
+     */
+    suspend fun mapUser(imsProject: IMSProject, userData: UserData?): User {
+        val databaseId = userData?.asUser()?.databaseId
+        if (databaseId != null) {
+            val encodedAccountId =
+                jsonNodeMapper.jsonNodeToDeterministicString(objectMapper.valueToTree<JsonNode>(databaseId))
+            val foundImsUser =
+                imsProject.ims().value.users().firstOrNull { it.templatedFields["github_id"] == encodedAccountId }
+            if (foundImsUser != null) {
+                return foundImsUser
+            }
+        } else {
+            val foundImsUser =
+                imsProject.ims().value.users().firstOrNull { it.username == (userData?.login ?: "github") }
```

Comment:
you use this "github" constant 5 times, maybe add a constant for this?

Reply:
9568d5bee85f0d3491c71a29a1908da728ff092c